packages/jupyter/package.json

```diff
@@ -29,11 +29,11 @@
     "@scienceicons/react": "^0.0.6",
     "buffer": "^6.0.3",
     "classnames": "^2.3.2",
-    "myst-common": "^1.1.16",
-    "myst-config": "^1.1.16",
-    "myst-frontmatter": "^1.1.16",
+    "myst-common": "^1.1.17",
+    "myst-config": "^1.1.17",
+    "myst-frontmatter": "^1.1.17",
     "myst-spec": "^0.0.4",
-    "myst-spec-ext": "^1.1.16",
+    "myst-spec-ext": "^1.1.17",
```

Comment:
Can you ensure all other mystmd deps are also bumped! For example #286 as a template.